pom.xml

```diff
@@ -28,7 +28,7 @@
   <description>A Collection of Extensions for Syndesis</description>
 
   <properties>
-    <syndesis.version>1.3.10</syndesis.version>
+    <syndesis.version>1.4.0.fuse-000002</syndesis.version>
```

Comment:
I'm not sure about this version change, is this something we wish to do? The previous (I think) kept us in check that we remain compatible from the moment we introduced extensions.